application/forms/PreferenceForm.php

```diff
@@ -208,11 +209,16 @@ public function createElements(array $formData)
                     )
                 );
             }
+        } else {
+            $themeFile = StyleSheet::getThemeFile(Config::app()->get('themes', 'default'));
```

Comment:
Hm, I was too fast. This runs only if the user cannot change the theme. But that shouldn't affect the mode selection anyway. Already initialize `$themeFile` with the global default theme. Don't do it here.

Reply:
I like that even better